epsagon/trace.py

```diff
@@ -939,6 +946,16 @@ def _send_traces(self):
         """
         if self.token == '' or self.trace_sent:
             return
+        if self.min_error_code and self.runner:
```

Comment:
Move all this logic into a different function (and please make sure it is fail-safe)

Reply:
Also please add a small UT for that